src/meta/processors/admin/SnapShot.cpp

```diff
@@ -63,9 +67,13 @@ cpp2::ErrorCode Snapshot::blockingWrites(storage::cpp2::EngineSignType sign) {
     auto spacesHosts = retSpacesHosts.value();
     for (const auto& spaceHosts : spacesHosts) {
         for (const auto& host : spaceHosts.second) {
+            LOG(INFO) << "will block write host: " << host;
             auto status = client_->blockingWrites(spaceHosts.first, sign, host).get();
             if (!status.ok()) {
                 LOG(ERROR) << " Send blocking sign error on host : " << host;
+                if (sign == storage::cpp2::EngineSignType::BLOCK_ON) {
```

Comment:
This is because when deleting the write flag, we need to delete as many flags as possible.

Reply:
You said that 
> This is because in setWriteBlocking, it goes through all the parts and then sets the block flag, and finally sends a sync, where the sync must have a part of the failure (because the leader change), and I think this part of the failure can be ignored, because the failure of the machine will eventually call setWriteBlocking in the meta.

I believe this is where meta call setWriteBlocking, I don't think we can ignore failure in any case (If you ignore failure when unblock, the part can't service anymore). We'd better modify `NebulaStore::setWriteBlocking`.

Reply:
> You said that
> 
> > This is because in setWriteBlocking, it goes through all the parts and then sets the block flag, and finally sends a sync, where the sync must have a part of the failure (because the leader change), and I think this part of the failure can be ignored, because the failure of the machine will eventually call setWriteBlocking in the meta.
> 
> I believe this is where meta call setWriteBlocking, I don't think we can ignore failure in any case (If you ignore failure when unblock, the part can't service anymore). We'd better modify `NebulaStore::setWriteBlocking`.

Okay, here I think we need to call setWriteBlocking as much as possible, but we need to keep the error return to the upper level.